homeassistant/components/toon/sensor.py

```diff
@@ -140,16 +149,6 @@ def state(self) -> str | None:
         )
         return getattr(section, SENSOR_ENTITIES[self.key][ATTR_MEASUREMENT])
 
-    @property
-    def unit_of_measurement(self) -> str | None:
-        """Return the unit this state is expressed in."""
-        return SENSOR_ENTITIES[self.key][ATTR_UNIT_OF_MEASUREMENT]
-
-    @property
-    def device_class(self) -> str | None:
```

Comment:
It it a good idea to remove the device_class ?

Reply:
it is not removed, it is now set as a class var, together with the others.

Reply:
thanks.